### ISSUE TITLE
Event for brackets open/close?

### ISSUE THREAD

**Issue by at185190** [leezng/vue-json-pretty#90]
So I've spent good eternity going through source code and trying to find a way to get an event for brackets close/open.
But so far I've got nowhere.

This is required to keep data in focus, for moving screen when json is collapsed or expanded you don't need to go far to see the issue, demo page - example 2, collapse Json, expand it and half won't be visible.

**Comment by leezng** [comment]
It means that you want a callback for brackets open/close? But I can’t see your usage scenario in the demo page.

**Comment by at185190** [comment]
Yes, I want callback event just like there is for data change and for click which is not click, its selection, that should be renamed to be honest.

As for usage, image I have some sort of data output div that I can scroll to make sure its always scrolled down and latest data is visible, which works fine, because I know when to scroll, new data comes , output scrolls to bottom. Closing opening bracket, makes a mess out of that, because while closing the bracket gives space, opening it doesn't, it takes away screen space.

![image](https://user-images.githubusercontent.com/69504978/96691148-72eded80-137c-11eb-8682-ce75cd85bda0.png)

See how in demo page there is a scroll bar? This is after I opened the brackets, half of the second json is not visible, now technically I want to see second json, not first, because I clicked on it, I would like the page to scroll and focus on the second json.
While it's a demo page and not particularly required to do that, considering there is functionality to collapse and expand json and emitting event that is capturable shouldn't be too hard.

While scrolling/focusing or doing anything with the screen or json is developers responsibility, we need a way to know when such action occurred.

I don't think it is required to return any data with collapse/expand event
